package Exception;

public class Use_of_throws {
	public static void main(String[] args)throws Throwable {
		System.out.println("Hii");
		Thread.sleep(2000);
		
		
	}

}
